app/controllers/api/irs_attempts_api_controller.rb

```diff
@@ -29,12 +30,32 @@ def create
             headers['X-Payload-Key'] = log_file_record.encrypted_key
             headers['X-Payload-IV'] = log_file_record.iv
 
-            requested_data = s3_helper.s3_client.get_object(
+            response = s3_helper.s3_client.head_object(
```

Comment:
is this something we'd want to guard on a feature flag? Just in case we run into issues with like, NGINX in a deployed env or something that has issues with live responses?

Reply:
Implemented :)